### PR TITLE
Use the correct path prefix in the images loader

### PR DESCRIPTION
**Pull request by leofeyer**
To reproduce, add a preview image to a theme in the back end.

### CODE REVIEW

**Comment by zoglo** [comment]
@leofeyer just tested it - This breaks all other preview images 🙈 (e.g. file preview)

**Comment by zoglo** [comment]
Works after the latest commit.

File: core-bundle/src/InsertTag/Resolver/LegacyInsertTag.php [hunk 1]
```diff
@@ -591,8 +591,8 @@ public function __invoke(ResolvedInsertTag $insertTag): InsertTagResult
                         $picture = $this->container->get('contao.image.picture_factory')->create($this->container->getParameter('kernel.project_dir').'/'.$strFile, $size);
 
                         $data = [
-                            'img' => $picture->getImg($this->container->getParameter('kernel.project_dir'), $staticUrl),
-                            'sources' => $picture->getSources($this->container->getParameter('kernel.project_dir'), $staticUrl),
+                            'img' => $picture->getImg($this->container->getParameter('contao.web_dir'), $staticUrl),
+                            'sources' => $picture->getSources($this->container->getParameter('contao.web_dir'), $staticUrl),
```
Comment (zoglo) [inline core-bundle/src/InsertTag/Resolver/LegacyInsertTag.php:595]:
```suggestion
                            'img' => $picture->getImg($this->container->getParameter('kernel.project_dir'), $staticUrl),
                            'sources' => $picture->getSources($this->container->getParameter('kernel.project_dir'), 
```
This breaks the picture insert tag.
Reverting this change makes the `{{picture::*}}` inserttag render correctly in the front-end

**Review by zoglo — CHANGES_REQUESTED** [review]

**Comment by zoglo** [comment]
The latest change also breaks every image in the frontend.

The previous URLs that was output:
`localhost/assets/images/a/your-image.jpg`

The new one:
`localhost/public/assets/images/a/your-image.jpg` -> 404

**Comment by leofeyer** [comment]
We have to remove the feature again, I‘m afraid: #7406

**Pull request closed without merging**